src/NServiceBus.NHibernate.TransactionalSession.AcceptanceTests/EndpointConfigurer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting.Support;
+
+public abstract class EndpointConfigurer : IConfigureEndpointTestExecution
```

Comment:
Forgive my ignorance. Why was this abstraction needed?

Reply:
This class was taken `as-is` from the main package's atts. I'm pretty sure `abstract` is not needed but wanted to make sure the code is as close to each other as possible.